src/optimize/base_optimizer.js

```diff
@@ -212,15 +213,13 @@ export default class BaseOptimizer {
      * of Kibana and just make compressing and extracting it more difficult.
      */
     const maybeAddCacheLoader = (cacheName, loaders) => {
-      if (IS_KIBANA_DISTRIBUTABLE) {
-        return loaders;
-      }
-
       return [
         {
           loader: 'cache-loader',
           options: {
-            cacheDirectory: this.uiBundles.getCacheDirectory(cacheName)
+            cacheContext: fromRoot('.'),
+            cacheDirectory: this.uiBundles.getCacheDirectory(cacheName),
+            readOnly: !!IS_KIBANA_DISTRIBUTABLE && !(process && process.env && process.env.FORCE_WRITABLE_CACHE)
```

Comment:
No need to check for `process && process.env`, those are globals that will always be defined in Node.

Reply:
👍